incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -43,69 +43,73 @@ spec:
-{{ toYaml .Values.collector.resources | indent 10 }}
+        - name: {{ template "jaeger.collector.name" . }}
+          securityContext:
+              {{- toYaml .Values.collector.securityContext | nindent 12 }}
+          image: {{ .Values.collector.image }}:{{ .Values.tag }}
+          imagePullPolicy: {{ .Values.collector.pullPolicy }}
+          env:
+            {{- range $key, $value := .Values.collector.cmdlineParams }}
+            - name: {{ $key | replace "." "_" | replace "-" "_" | upper | quote }}
+              value: {{ $value }}
+            {{- end }}
+            - name: SPAN_STORAGE_TYPE
+              value: {{ .Values.storage.type }}
+            {{- if eq .Values.storage.type "cassandra" }}
+            - name: CASSANDRA_SERVERS
+              value: {{ template "cassandra.host" . }}
+            - name: CASSANDRA_PORT
+              value: {{ .Values.storage.cassandra.port | quote }}
+            - name: CASSANDRA_KEYSPACE
+              value: {{ printf "%s_%s" "jaeger_v1" .Values.cassandra.config.dc_name | quote }}
+            - name: CASSANDRA_USERNAME
+              value: {{ .Values.storage.cassandra.user }}
+            - name: CASSANDRA_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ if .Values.storage.cassandra.existingSecret }}{{ .Values.storage.cassandra.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-cassandra{{- end }}
+                  key: password
+            {{- end }}
+            {{- if eq .Values.storage.type "elasticsearch" }}
+            - name: ES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ if .Values.storage.elasticsearch.existingSecret }}{{ .Values.storage.elasticsearch.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-elasticsearch{{- end }}
+                  key: password
+            - name: ES_SERVER_URLS
+              value: {{ template "elasticsearch.client.url" . }}
+            - name: ES_USERNAME
+              value: {{ .Values.storage.elasticsearch.user }}
+            {{- end }}
+          ports:
+          - containerPort: {{ .Values.collector.service.grpcPort }}
+            name: grpc
+            protocol: TCP
+          - containerPort: {{ .Values.collector.service.tchannelPort }}
+            name: tchannel
+            protocol: TCP
+          - containerPort: {{ .Values.collector.service.httpPort }}
+            name: http
+            protocol: TCP
+          - containerPort: 14269
+            name: admin
+            protocol: TCP
+          - containerPort: {{ .Values.collector.service.zipkinPort }}
+            name: zipkin
+            protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /
+              port: admin
+          livenessProbe:
+            httpGet:
+              path: /
+              port: admin
+          resources:
+            {{ toYaml .Values.collector.resources | indent 12 }}
```

Comment:
Why was this indentation increased to 12? Am I missing something? Please, revert.

Reply:
The indentation change is unrelated to security context. Please, revert it and keep one logical change per PR.

Reply:
I will wait to see if you let this one slide... 1 logical change per PR is definitely the way to go and I will strive for this moving forward. Just seeing that you took the time to go over the changes, maybe we could let this one slide :)